### PR TITLE
The Calculator Homework

### PR DESCRIPTION
**Pull request by RobertStoelhorst**
Hi Camilla

Enjoy!

### CODE REVIEW

**Comment by RobertStoelhorst** [comment]
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
A: 8

Was there anything that you struggled with?;
A: the last line in part 2: was difficult I had to sleep on it.

Is there anything that you'd like some further information on?;
A: in terminal there was something about copy and pasting each day Joel went over I cant quite remember what it was, I will raise it next time it comes up.

Roughly how long did it take?
a good 4 hours to get it clean enough that I was satisfied.

File: Robert Stoelhorst/week01/day_03/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title>Javascript Exercises</title>
+    <meta charset="UTF-8">
+  </head>
+
+  <body>
+    <p>Please check the Javascript console for program output.</p>
+    <script src="functions/homework.js"></script>
```
Comment (CamillaCdC) [inline Robert Stoelhorst/week01/day_03/index.html:9]: this script tag should be in your <head> tags.

File: Robert Stoelhorst/week01/homework.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title>Javascript Exercises</title>
+    <meta charset="UTF-8">
+  </head>
+
+  <body>
+    <p>Please check the Javascript console for program output.</p>
+    <script src="functions/homework.js"></script>
```
Comment (CamillaCdC) [inline Robert Stoelhorst/week01/homework.html:9]: same as above

**Comment by CamillaCdC** [comment]
Sleeping on it is an awesome way to get un-stuck! Part 2 was meant to be difficult.
Your code looks good! Please keep in mind indentation and spacing. In some spots you nail it, and it others its a little inconsistent. 

Hopefully what Joel went over this morning with GitHub helped sort out that issue you were having. Let me know in Slack if you're still having trouble and we can sort it out together.

**Pull request merged**